compiler/resolution/functionResolution.cpp

```diff
@@ -7626,16 +7626,60 @@ static void lvalueCheckActual(CallExpr* call, Expr* actual, IntentTag intent, Ar
   }
 }
 
+// heuristic to walk up a series of temps to find the base object of a method call
+static Symbol* maybeGetBaseSym_helper(SymExpr* def);
```

Comment:
`..._helper`, the mid-name underscore, is unaesthetic. Do we do this in Chapel in other places? If not, could we pick a different name?

Reply:
It gets used in a few places, but the camel-case version `...Helper` is more common. I will rename it